### ISSUE TITLE
update README with run instructions

### ISSUE THREAD

**Issue by andrewkern** [popsim-consortium/analysis2#41]
At this point, if the design has settled down, I think we could use some more details in the README about how to run things.

In particular we will want:

- details on the config file
- instructions on how to run Snakemake (e.g. command line examples)
- details on expected output

what else?

**Comment by xin-huang** [comment]
How to get external tools

- `polyDFE`: `git clone https://github.com/paula-tataru/polyDFE`
- `DFE-alpha`: 
    - Download DFE-alpha: http://www.homepages.ed.ac.uk/pkeightl/dfe_alpha/download-dfe-alpha.html
    - Download DFE-alpha data tables: https://datashare.ed.ac.uk/handle/10283/2730
- `grapes`: 
    - `wget -c https://github.com/BioPP/grapes/releases/download/v1.1.0/grapes-x86_64-bin-static-1.1.0-1.tar.gz`
    - `tar -xvf grapes-x86_64-bin-static-1.1.0-1.tar.gz`

**Comment by andrewkern** [comment]
i'm moving some of this external tool stuff into the snakemake to make install easier, and I notice that `dfe-alpha` requires a 5GB download!!

This isn't great. We should come up with a better way to store / serve up this file

**Comment by andrewkern** [comment]
update here: #60 has the features i mentioned above. 

also i've mirrored the large file for `dfe-alpha` on our server at UO -- i'm hoping that will speed downloads?

**Comment by xin-huang** [comment]
I think we should let Peter Keightley know and request permission from him if we mirror the data file for `dfe-alpha`?

**Comment by andrewkern** [comment]
yes- good idea. let's see if it actually makes a difference in download speed first, maybe?
